fault/verilator_target.py

```diff
@@ -589,9 +594,17 @@ def make_file_read(self, i, action):
         ])
 
     def make_var(self, i, action):
-        if isinstance(action._type, AbstractBitVectorMeta) and \
-                action._type.size == 32:
-            return [f"unsigned int {action.name};"]
+        if isinstance(action._type, AbstractBitVectorMeta):
```

Comment:
I think something like this would work:
```python
if issubclass(action._type, SIntVector):
    <signed code>
elif issubclass(action._type, BitVector):  # works for both BitVector and UIntVector
    <unsigned code>
raise NotImplementedError
```